src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjects.cs

```diff
@@ -29,15 +29,21 @@ public GetProjects(IProjectsService projectService)
             try
             {
                 var projects = this.projectService.GetProjects();
+                if (projects == null)
+                {
+                    throw new ProjectsException(ProjectsErrors.NotFound);
+                }
+
                 return new OkObjectResult(projects);
             }
             catch (ProjectsException e)
             {
-                return new ContentResult
-                {
-                    StatusCode = e.StatusCode,
-                    Content = e.ProjectsErrorMessage,
-                };
+                return e.Error;
+            }
+            catch (System.Exception e)
+            {
+                var errorException = new ProjectsException(ProjectsErrors.InternalServerError, e);
+                return errorException.Error;
```

Comment:
I see that in GetProjectById you have used
var errorException = new ProjectsException(ProjectsErrors.InternalServerError, ex);
var error = new ObjectResult(errorException.ErrorMessage);
error.StatusCode = errorException.StatusCode;
return error;

is the difference in how to handle that intended? I mean, all 500 (Exception e) should be handled the same way from my point of view

Reply:
This is the correct one, my bad, fixing GetProjectById

Reply:
done